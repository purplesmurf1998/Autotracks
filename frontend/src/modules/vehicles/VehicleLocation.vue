<template>
  <div>
    <CCard>
      <CCardBody>
        <h4>Location</h4>
        <hr />
      </CCardBody>
    </CCard>
  </div>
</template>

<script>
export default {
  name: 'VehicleLocation',
  props: ["vehicle"],
};
</script>

<style>
</style>