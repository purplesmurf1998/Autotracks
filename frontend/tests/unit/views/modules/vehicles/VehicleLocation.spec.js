import {createLocalVue, shallowMount} from '@vue/test-utils'
import CoreuiVue from '@coreui/vue-pro'
import Vuex from 'vuex'

import VehicleLocation from '../../../../../src/modules/vehicles/VehicleLocation.vue'

const localVue = createLocalVue()
localVue.use(CoreuiVue)
localVue.use(Vuex)

const store = new Vuex.Store({
    state: {
      sidebarShow: 'responsive',
      sidebarMinimize: false,
      asideShow: false,
      darkMode: false
    }
  })

  describe('VehicleLocation.vue', () => {
    const wrapper = shallowMount(VehicleLocation, { 
      store, 
      localVue
    })
    it('has a name', () => {
      expect(VehicleLocation.name).toBe('VehicleLocation')
    })
    it('should render correct content', () => {
        expect(wrapper.find('h4').text()).toMatch('Location')
    })
    test('renders correctly', () => {
      expect(wrapper.element).toMatchSnapshot()
    })
  })