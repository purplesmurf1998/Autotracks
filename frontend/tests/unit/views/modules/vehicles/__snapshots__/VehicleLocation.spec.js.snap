// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VehicleLocation.vue renders correctly 1`] = `
<div>
  <ccard-stub>
    <ccardbody-stub>
      <h4>
        Location
      </h4>
       
      <hr />
    </ccardbody-stub>
  </ccard-stub>
</div>
`;
